languages/cpp/reference/implementing-a-concept-exercise.md

```diff
@@ -1,6 +1,12 @@
 # How to implement a C++ concept exercise
 
-This document describes how to implement a concept exercise for the C++ track. As this document is generic, the following placeholders are used:
+This document describes how to implement a concept exercise for the C++ track. Before diving into the implementation, please read up on the following documents:
+
+- [The features of v3][docs-features-of-v3].
+- [Rationale for v3][docs-rationale-for-v3].
+- [What are concept exercise and how they are structured?][docs-concept-exercises]
```

Comment:
The links are missing for this file.

Reply:
Weird. I would have expected the link checker to verify this. Thanks for catching it.